lib/ReplaceSource.js

```diff
@@ -162,6 +162,7 @@ class ReplaceSource extends Source {
 					? sourceContents[sourceIndex]
 					: undefined;
 			if (content === undefined) return false;
+			if (content === null) return false;
```

Comment:
Can `content` be `null`?

Reply:
Yes, it might happened in v3.0.2, but already fixed by 3.1.1, I will make a new commit to make this file stay untouched.